lib/ledger_sync/adaptors/ledger_serializer_attribute_set.rb

```diff
@@ -32,7 +32,7 @@ def add(attribute)
         end
 
         if attribute.resource_attribute.present?
-          raise "resource_attribute already defined for #{serializer_class.name}: #{attribute.resource_attribute}" if resource_attribute_keyed_hash.key?(attribute.resource_attribute.to_s)
+          # raise "resource_attribute already defined for #{serializer_class.name}: #{attribute.resource_attribute}" if resource_attribute_keyed_hash.key?(attribute.resource_attribute.to_s)
```

Comment:
I had to remove this line, as expected.  This PR also largely ignores that we need to split the serializers up for this adaptor.  We will need a serializer and deserializer, which are both essentially intelligent "mergers."